src/components/TextWithSelectField/TextWithSelectField.tsx

```diff
@@ -92,15 +76,14 @@ const TextWithSelectField: React.FC<TextWithSelectFieldProps> = ({
               name={selectFieldName}
               onChange={value => change({ target: { name: selectFieldName, value } })}
               value={selectFieldValue}
-              className={clsx("noBorder", selectFieldClassName)}
+              className="noBorder"
```

Comment:
suggestion: I'd try to get rid of this class and style  Input directly as it possible no.

Reply:
Ok I see, there is wrong implementation in Select. Leave it as is.